### ISSUE TITLE
1.7.0: httpdomain not redy for sphinx 4.0.2

### ISSUE THREAD

**Issue by kloczek** [sphinx-contrib/httpdomain#48]
```console
+ sphinx-build -b man -d python-sphinxcontrib-httpdomain doc .
Running Sphinx v4.0.2

Extension error:
Could not import extension sphinxcontrib.autohttp.flask (exception: cannot import name 'force_decode' from 'sphinx.util' (/usr/lib/python3.8/site-packages/sphinx/util/__init__.py))
```

**Comment by kloczek** [comment]
pytest is failing as well
```console
+ /usr/bin/pytest -ra
=========================================================================== test session starts ============================================================================
platform linux -- Python 3.8.9, pytest-6.2.4, py-1.10.0, pluggy-0.13.1
benchmark: 3.4.1 (defaults: timer=time.perf_counter disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)
rootdir: /home/tkloczko/rpmbuild/BUILD/httpdomain-1.7.0
plugins: forked-1.3.0, shutil-1.7.0, virtualenv-1.7.0, expect-1.1.0, httpbin-1.0.0, flake8-1.0.7, timeout-1.4.2, betamax-0.8.1, freezegun-0.4.2, case-1.5.3, isort-1.3.0, aspectlib-1.5.2, asyncio-0.15.1, toolbox-0.5, xprocess-0.17.1, aiohttp-0.3.0, checkdocs-2.7.0, mock-3.6.1, rerunfailures-9.1.1, requests-mock-1.9.3, cov-2.12.1, pyfakefs-4.5.0, cases-3.6.1, flaky-3.7.0, hypothesis-6.14.0, benchmark-3.4.1, xdist-2.3.0, Faker-8.8.1
collected 2 items / 1 error / 1 selected

================================================================================== ERRORS ==================================================================================
___________________________________________________________________ ERROR collecting test/bottle_test.py ___________________________________________________________________
ImportError while importing test module '/home/tkloczko/rpmbuild/BUILD/httpdomain-1.7.0/test/bottle_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib64/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
test/bottle_test.py:3: in <module>
    from sphinxcontrib.autohttp.bottle import get_routes
/usr/lib/python3.8/site-packages/sphinxcontrib/autohttp/bottle.py:20: in <module>
    from sphinx.util import force_decode
E   ImportError: cannot import name 'force_decode' from 'sphinx.util' (/usr/lib/python3.8/site-packages/sphinx/util/__init__.py)
========================================================================= short test summary info ==========================================================================
ERROR test/bottle_test.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
============================================================================= 1 error in 0.31s =============================================================================
```

**Comment by tuukkamustonen** [comment]
Duplicate of https://github.com/sphinx-contrib/httpdomain/issues/46

**Comment by Zlopez** [comment]
This could be probably closed, because #46 is already closed.

**Comment by kloczek** [comment]
Yep just tested 1.8.0 and all looks fine.
Closing.

Thank you.
